e2e/_support/auth.ts

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { APIRequestContext, Browser, BrowserContext, expect, Page } from "@playwright/test";
+import { callBackend } from "./backend";
+
+export type User = {
```

Comment:
This allows us to use IntelliSense on the user object, i.e. see the available attributes. In the future, we could something like this for important objects from the FactoryBot as well, e.g. lecture...